pkg/kritis/admission/admission.go

```diff
@@ -306,7 +307,10 @@ func checkBreakglass(meta *metav1.ObjectMeta) bool {
 }
 
 // TODO: update this once we have more metadata clients
```

Comment:
Remove this TODO.
Shd we use the same function for getting the client in cron.go and admission.go and remove the code duplication.

Reply:
@tejal29 good suggestion. I updated the PR.